readme-sync/v0/senseml-reference/8000 - concepts/1800 - jsonlogic.md

```diff
@@ -48,25 +49,23 @@ Returns a boolean to indicate if the specified value exists.
 
 ```json
 {
-    "exists": [
-        JsonLogic
-    ]
+    "exists": JsonLogic
 }
 ```
 
-Most commonly used with the JsonLogic `var`  operation to test that an output value isn't null. The  `var` operation retrieves extracted field values using field `id` keys. 
+Most commonly used with the JsonLogic `var` operation to test that an output value isn't null. Returns true if the value being evaluated is neither null nor undefined. Can take an array (where it will check the first item only) or a single value (e.g., `{ "exists": [{ "var": "some_field" }] }` or `{ "exists": { "var": "some_field" } }`).
 
 ### Examples
 
 See [Validating extractions](doc:validate-extractions#examples).
 
 ## Flatten
 
-Takes as input an array that can contain nested arrays, and returns a single-level array populated with the same values.  This operation is similar to the built-in JsonLogic [merge](https://jsonlogic.com/operations.html#merge) operation, except that it's recursive to any depth. 
+Takes as input an array that can contain any depth of nested arrays, and returns a single-level array populated with the same values.
 
 ### Examples
 
-The following example shows that the Flatten operation's output varies depending on the context.
```

Comment:
Changed this so we're not incorrectly suggesting that flatten/JsonLogic itself works differently in different contexts - they work the same, but the different contexts (custom comp vs. anywhere else) affect whether the output gets another layer of processing.